native/cocos/renderer/gfx-vulkan/VKGPUObjects.h

```diff
@@ -184,61 +201,64 @@ struct CCVKGPUBuffer {
 
     // descriptor infos
     VkBuffer vkBuffer = VK_NULL_HANDLE;
-    VkDeviceSize startOffset = 0U;
     VkDeviceSize size = 0U;
 
     VkDeviceSize instanceSize = 0U; // per-back-buffer instance
-    ccstd::vector<ThsvsAccessType> currentAccessTypes;
+    mutable ccstd::vector<ThsvsAccessType> currentAccessTypes;
```

Comment:
mutable member usually means thread-safety. See [https://softwareengineering.stackexchange.com/questions/274548/const-means-thread-safe](url)
Consider adding mutex, or change member function constness.

Reply:
I have chaned constness of ```TextureView``` and ```BufferView```. 

Here are 3 members should be removed later, which still has mutable specifier.
a.  ```vector<uint32_t> clearValues``` in CCVKGPURenderpass.
b. ```vector<uint32_t> dynamicOffsetOffsets``` in CCVKGPUPipelineLayout.
c. ```vector<VkBuffer> recordedBuffers``` in CCVKGPUCommandBuffer.